src/sentry/features/base.py

```diff
@@ -62,3 +63,14 @@ def __init__(self, name: str, user: User) -> None:
 
     def get_subject(self) -> User:
         return self.user
+
+
+class FeatureHandlerStrategy(Enum):
+    """
+    This controls whether the feature flag is evaulated statically,
+    or if it's managed by a feature flag service.
+    See https://develop.sentry.dev/feature-flags/
+    """
+
+    STATIC = 1
```

Comment:
Not all other feature flags are managed statically. We have lots that are managed via options that are dynamic but not flagr

Reply:
@wedamija good point. maybe `External` v.s. `Internal`? open to ideas

Reply:
`Internal` and `Remote` 🤷 `External` works too.